### PR TITLE
12.0 kpd 857 & KPD-894

### PR DESCRIPTION
**Pull request by SebastianGieser-Vertel**

### CODE REVIEW

File: calendar_af/wizard/create_local_occasion.py [hunk 1]
```diff
@@ -220,6 +220,8 @@ def _action_create_occasions(self):
                 repeat_list.append(3)
             if self.repeat_fri:
                 repeat_list.append(4)
+            if not (self.repeat_mon or self.repeat_tue or self.repeat_wed or self.repeat_thu or self.repeat_fri):
```
Comment (nils-vertel) [inline calendar_af/wizard/create_local_occasion.py:223]: easier to write if not repeat_list: ?

Comment (nils-vertel) [reply calendar_af/wizard/create_local_occasion.py:223]: Maybe think if its possible that people work on sat/sun?

**Review by nils-vertel — APPROVED** [review]
Minor things, check them and decide if you want to fix it

File: calendar_af/wizard/create_local_occasion.py (hunk 1 above)
Comment (SebastianGieser-Vertel) [reply calendar_af/wizard/create_local_occasion.py]: Changed it to repeat_list, we don't allow the creation of occasions on sat/sun at the moment.

**Review by nils-vertel — APPROVED** [review]

**Pull request merged**